clis/tcli/src/cli.rs

```diff
@@ -318,48 +322,20 @@ macro_rules! cli_try {
     };
 }
 
-fn custom_format(
-    w: &mut dyn std::io::Write,
-    now: &mut DeferredNow,
-    record: &Record,
-) -> Result<(), std::io::Error> {
-    write!(
-        w,
-        "{} {} {} {:#?}:{:#?} {} {}",
-        now.now().month(),
-        now.now().day(),
-        now.now().time(),
-        record.module_path().unwrap_or("unknown module"),
-        record.line().unwrap_or(0),
-        record.level(),
-        record.args()
-    )
-}
-
 impl Cli {
     pub fn new(
         features: Features,
         token: Option<String>,
         derp_server: Arc<Mutex<Option<Server>>>,
     ) -> anyhow::Result<Self> {
-        let base_name = "tcli";
-        let suffix = "log";
-        if let Ok(logger) = Logger::try_with_str("debug") {
-            if let Err(err) = logger
-                .log_to_file(
-                    FileSpec::default()
-                        .basename(base_name)
-                        .suppress_timestamp()
-                        .suffix(suffix),
-                )
-                .format(custom_format)
-                .write_mode(WriteMode::BufferAndFlush)
-                .use_utc()
-                .start()
-            {
-                error!("{}", err);
-            }
-        }
+        let (non_blocking_writer, _tracing_worker_guard) =
+            tracing_appender::non_blocking(File::create("tcli.log")?);
+        tracing_subscriber::fmt()
+            .with_writer(non_blocking_writer)
+            .with_ansi(false)
+            .with_line_number(true)
+            .with_level(true)
+            .init();
```

Comment:
The module information is printed by default. The ".with_file(true)" function enables actual file path to be printed which was not present in the original logger.
I couldn't find a convenient way to format the time logging, but the current format looks like this: `2023-12-01T08:18:14.160963Z`. I could implement a custom tracing subscriber so that the log format is identical to the old one, but that would take time and I don't know if it's necessary.